src/shared/use-context-selection/use-context-selection.js

```diff
@@ -27,6 +27,20 @@ export function usePeriodId() {
     return useQueryParam('periodId', PARAMS_SCHEMA.periodId)
 }
 
+export function useFeature(feature) {
```

Comment:
I added this "feature toggle" in order to be able to test the functionality without the backend and maintenance app deployed.

Reply:
This makes it easy for testing 😅, but I'm not sure we actually would want to expose a query parameter like this? I think people who design data sets would not want their users to arbitrarily transpose things (I assume we're not going to advertise this, but I guess it's possible that people share links if we expose it?)

There aren't really checks for this, so providing invalid categories causes some issues, but maybe that's not of concern if it's just for testing?